### PR TITLE
Newcomers_Guide: Mention newcomer trial process

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -3,15 +3,20 @@
...
+to complete. The newcomer process is as it follows:
 
-There's a few things a contributor needs in order to be considered a full-norm
-coalaian:
+You will start as a newcomer, which is kind of a trial. If you complete the
+following tasks, you will become a developer at coala:
 
   - merge a ``difficulty/newcomer`` Pull Request
-  - review other Pull Requests
+  - review at least a ``difficulty/newcomer`` Pull Request
```
Comment (sils) [inline Developers/Newcomers_Guide.rst]: let's make it three, every newcomer PR takes probably three iterations

Comment (sils) [inline Developers/Newcomers_Guide.rst]: ah didn't see line 15, all good

File: Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -3,15 +3,20 @@
...
   - merge a ``difficulty/low`` Pull Request
+  - review at least a``difficulty/low`` or higher Pull Request
+
+Once you have achieved all these and the maintainers consider that you have
+helped with reviewing as it is stated, you will be promoted to the developers
+team.
```
Comment (sils) [inline Developers/Newcomers_Guide.rst]: maybe something like "Just ask for being promoted on the chat and provide links to your reviews and merged Pull Requests!"

Also we should add a benefit for being a developer. Maybe rephrase the last part of this sentence to "you will be able to name yourself a coala developer", it's kind of "now you're accepted" but polite

**Comment by sils** [comment]
ack https://github.com/coala/documentation/pull/198/commits/848373df782bc2525cdbaffe1294966306da5888

Why the hell was this working anyway?

**Comment by sils** [comment]
ack 848373d

**Comment by Adrianzatreanu** [comment]
reack c999c08

**Comment by sils** [comment]
please reorder your commits for gitmate :)

**Comment by sils** [comment]
ack https://github.com/coala/documentation/pull/198/commits/37006fa7a97817c202c667da3d7b75d72dba6909 https://github.com/coala/documentation/pull/198/commits/9b04b404793124c06643a24ea82c5b8254a0e448 https://github.com/coala/documentation/pull/198/commits/fd53335789bc6e39c37a4e17079b5b71e816513f

**Comment by sils** [comment]
@rultor merge
